OpenLaTeXStudio/EditorModule/nbproject/project.xml

```diff
@@ -105,6 +105,14 @@
                     </run-dependency>
                 </dependency>
                 <dependency>
+                    <code-name-base>org.openide.util.ui</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <specification-version>9.4.1</specification-version>
+                    </run-dependency>
+                </dependency>
+                <dependency>

```

Comment:
I'm guessing this is because you are working on NB8.0.2 (as per the file
path) while my version is NB8.1. Apparently, stuff was moved between
libraries between these versions -- this seems to me to be the most likely
cause of the build error.

Cf. https://forums.netbeans.org/topic64808.html for a very similar issue.

On 20 March 2016 at 13:14, Sebastian Brudziński <notifications@github.com>
wrote:

> In OpenLaTeXStudio/EditorModule/nbproject/project.xml
> <https://github.com/sebbrudzinski/Open-LaTeX-Studio/pull/82#discussion_r56763346>
> :
>
> > @@ -105,6 +105,14 @@
> >                      </run-dependency>
> >                  </dependency>
> >                  <dependency>
> > +                    <code-name-base>org.openide.util.ui</code-name-base>
> > +                    <build-prerequisite/>
> > +                    <compile-dependency/>
> > +                    <run-dependency>
> > +                        <specification-version>9.4.1</specification-version>
> > +                    </run-dependency>
> > +                </dependency>
> > +                <dependency>
>
> I'm getting the following when attempting to build the project with those
> changes, which I assume is related with this new dependency:
>
> C:\Program Files\NetBeans 8.0.2\harness\suite.xml:187: The following error occurred while executing this line:
> C:\Program Files\NetBeans 8.0.2\harness\build.xml:174: No dependent module org.openide.util.ui
> BUILD FAILED (total time: 2 seconds)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/sebbrudzinski/Open-LaTeX-Studio/pull/82/files/89719b0140aed883ec5fefb458cdf0bb03ea62d0#r56763346>
>